foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/dao/PropertyDao.java

```diff
@@ -0,0 +1,150 @@
+package com.boozallen.aissemble.configuration.dao;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Configuration::Store
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.bettercloud.vault.Vault;
+import com.bettercloud.vault.VaultConfig;
+import com.bettercloud.vault.VaultException;
+import com.bettercloud.vault.api.mounts.MountPayload;
+import com.bettercloud.vault.api.mounts.MountType;
+import com.bettercloud.vault.response.LogicalResponse;
+import com.bettercloud.vault.response.MountResponse;
+import com.boozallen.aissemble.configuration.exception.PropertyDaoException;
+import com.boozallen.aissemble.configuration.store.Property;
+import com.boozallen.aissemble.data.encryption.VaultUtil;
+import com.boozallen.aissemble.data.encryption.config.DataEncryptionConfiguration;
+import org.aeonbits.owner.KrauseningConfigFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class PropertyDao implements Dao<Property> {
+    private static final Logger logger = LoggerFactory.getLogger(PropertyDao.class);
+
+    @Override
+    public boolean checkEmpty() {
+        return false;
```

Comment:
Q: I'm not familiar with the full interface, but what is the logic behind always returning false here?

Reply:
yeah.. should have put a comment but this function is not implemented yet so for now, it only returns false.

Reply:
Ok, given that - you could add a comment or throw an "unimplemented exception"